boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -57,6 +57,20 @@
 			pwms = <&ftm3 5 15625000 PWM_POLARITY_INVERTED>;
 		};
 	};
+
+	tja1042_0: can-transceiver0 {
```

Comment:
Intuitively, I would have chosen the naming in the opposite way: The specific manufacturer device type `tja1042` as the node name and the more generic `can-transceiver0` as the label. Maybe even (ab)use the address for numbering the different devices of same type.
```suggestion
	can-transceiver0: tja1042@0 {
```

This way the reference in the CAN controller does not change if the type / compatible of the transceiver changes.

Consider this just a remark. Definitely not blocking.

Reply:
Thanks. I have changed these to use the DTS nodelabel/name scheme applied in upstream Linux DTS. The devicetree specification does not (yet) mention CAN transceivers, but this naming scheme matches that used for USB and Ethernet PHYs.